dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/bnd.bnd

```diff
@@ -28,4 +28,5 @@ instrument.classesExcludes: org/apache/cxf/common/jaxb/*.class
   com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
   com.ibm.websphere.org.osgi.service.component;version=latest,\
   com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+  com.ibm.websphere.org.osgi.service.cm;version=latest,\
```

Comment:
Pulling in the CXF Activator as the cxf.core.3.2's Bundle-Activator (as you can see below) required another osgi dependency to compile